site/gatsby-site/src/tailwind.css

```diff
@@ -48,6 +48,34 @@
         @apply tw-flex-0-0-auto tw-w-auto tw-max-w-none;
     }
 
+    .tw-nav-header-container {
+        @apply tw-flex tw-flex-row tw-justify-between tw-items-center tw-w-full;
+    }
+
+    .tw-nav-header-container .nav-bar-header {
+        @apply tw-order-[0];
+    }
+
+    .tw-nav-header-container .divider {
+        @apply tw-w-[1px];
+    }
+
+    .tw-nav-header-container .nav-bar-brand {
+        @apply tw-items-center tw-flex tw-p-0;
+    }
+
+    .tw-nav-header-container .nav-bar-brand > * {
+        @apply tw-shrink-0;
+    }
+
+    .tw-nav-header-container .nav-bar-brand > .headerTitle {
+        @apply tw-shrink;
+    }
+
+    .tw-skip-to-content {
+        @apply tw-text-white tw-bg-gray-900 tw-relative tw-order-1 tw-ml-auto tw-opacity-0 tw-w-0 tw-h-0 tw-overflow-hidden focus:tw-opacity-100 focus:tw-py-[0ch] focus:tw-px-[1ch] 767px:tw-text-xs;
```

Comment:
I think the height and width need to be unset on focus. Currently when I tab-focus the skip to content link, it's invisible except for the focus outline in Firefox, and I can't focus it at all in Chromium.